File: src/backend/studio-reservation/src/main/java/org/taonaw/studio_reservation/usecase/command/studio/registerStudio/RegisterStudioCommand.java
package org.taonaw.studio_reservation.usecase.command.studio.registerStudio;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import org.taonaw.studio_reservation.domain.model.studio.EquipmentMaxUsableCount;
import org.taonaw.studio_reservation.domain.model.studio.StartTimes;
import org.taonaw.studio_reservation.domain.model.studio.StudioName;

import java.util.Set;

@Getter
@Builder
@AllArgsConstructor
public class RegisterStudioCommand {
    private StudioName studioName;
    private StartTimes startTimes;
    private Set<EquipmentMaxUsableCount> equipmentMaxUsableCounts;
}

File: src/backend/facility/src/main/kotlin/org/taonaw/facility/usecase/studio/registerStudio/RegisterStudioCommand.kt
package org.taonaw.facility.usecase.studio.registerStudio

import org.taonaw.facility.domain.model.studio.MaxRentalEquipmentQuantities
import org.taonaw.facility.domain.model.studio.StartTime
import org.taonaw.facility.domain.model.studio.StudioName
import org.taonaw.facility.domain.model.studio.StudioSize

data class RegisterStudioCommand(
        val studioName: StudioName,
        val studioSize: StudioSize,
        val startTime: StartTime,
        val maxRentalEquipmentQuantities: MaxRentalEquipmentQuantities) {
}